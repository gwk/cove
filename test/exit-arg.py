def f():
  exit(0)

f()
