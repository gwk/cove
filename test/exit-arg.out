----------------
Coverage Report:

__main__: exit-arg.py:
   1   def f():
   2     exit(0)
   3
   4 % f()

__main__: exit-arg.py: 4 lines; 1 trivial; 3 traceable; 2 covered; 0 ignored; 0 ignored but covered; 1 not covered.
